src/object.di

```diff
@@ -679,11 +679,18 @@ bool _xopCmp(in void* ptr, in void* ptr);
 void __ctfeWrite(T...)(auto ref T) {}
 void __ctfeWriteln(T...)(auto ref T values) { __ctfeWrite(values, "\n"); }
 
+///////////////////////////////////////
+
 template RTInfo(T)
 {
-    enum RTInfo = cast(void*)0x12345678;
+    enum RTInfo = __traits(getPointerBitmap,T).ptr;
```

Comment:
Ouch, the above actually allocates every time RTInfo!T is used in TypeInfo.rtInfo().